packages/contracts/tasks/helpers/benchmarks.ts

```diff
@@ -38,9 +38,9 @@ export async function publishBatch(
   let optimalBatchSize = batchSize;
```

Comment:
could we add the results to https://maci.pse.dev/docs/developers-references/smart-contracts/SupportedNetworks please?

Reply:
Oh, I did, I think forgot to include it in the last commit, let me upload it again